### ISSUE TITLE
same sentences with differnet text style in AnnotationCacheOperationSource section of reference docs

### ISSUE THREAD

**Issue by ethan3011** [spring-projects/spring-framework#32529]
**Affects:**  6.1.4

---

Hi again,

i just saw same sentences with differnet text style.

is there have different meaning? (like emphatic text has a parameter)

i just wanna to know but i can't find where can i ask about it.

thank you :)

target URL: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/cache/annotation/AnnotationCacheOperationSource.html#%3Cinit%3E(java.util.Set)

![image](https://github.com/spring-projects/spring-framework/assets/80470138/0c59b61b-55db-4331-9abd-8b26fef06467)

**Comment by ethan3011** [comment]
Hi again,

i think here also same letter with different style

thank you

![image](https://github.com/spring-projects/spring-framework/assets/80470138/5c9b5e3b-0bf6-4d1c-9159-07e4d98dd951)

**Comment by snicoll** [comment]
> is there have different meaning? (like emphatic text has a parameter)

No. If you look at [the code](https://github.com/spring-projects/spring-framework/blob/4b0a04857004e3d0e1040a1e70d6ac78af086a6f/spring-context/src/main/java/org/springframework/cache/annotation/AnnotationCacheOperationSource.java#L62), you can figure this out without having to ask a question here, which is not how we'd like to use the issue tracker, see [the guidelines for contributing](https://github.com/spring-projects/spring-framework/blob/master/CONTRIBUTING.md#discuss).

One has a `code` wrapper and the others don't. There is no reason behind this and it's probably a little inconsistent and will be polished.
